bson/bsoncodec/struct_tag_parser.go

```diff
@@ -117,3 +117,42 @@ var DefaultStructTagParser StructTagParserFunc = func(sf reflect.StructField) (S
 
 	return st, nil
 }
+
+// JsonFallbackStructTagParser has the same behavior as DefaultStructTagParser
+// but will also fallback to parsing the json tag instead on a field where the
+// bson tag isn't available.
+var JsonFallbackStructTagParser StructTagParserFunc = func(sf reflect.StructField) (StructTags, error) {
```

Comment:
I think this should be called JSONFallbackStructTagParser.

Reply:
done